pkg/agent/flowexporter/connections/connections.go

```diff
@@ -0,0 +1,134 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package connections
+
+import (
+	"sync"
+	"time"
+
+	"k8s.io/klog"
+
+	"github.com/vmware-tanzu/antrea/pkg/agent/flowexporter"
+	"github.com/vmware-tanzu/antrea/pkg/agent/interfacestore"
+	"github.com/vmware-tanzu/antrea/pkg/agent/openflow"
+)
+
+var _ ConnectionStore = new(connectionStore)
+
+type ConnectionStore interface {
```

Comment:
was there a reason for making this an interface?

Reply:
Thanks for pointing this. Yes, this can be a struct as multiple structs may not be there to implement this interface. Let me think about it more to make sure.

Reply:
There is a possibility of an integration test involving ConnectionStore  mock with flow records patch: https://github.com/vmware-tanzu/antrea/pull/825

I would like to leave this as interface.

Reply:
Happy to leave it as an interface if needed for testing